### PR TITLE
Issue-12: Created presentation model

### PR DESCRIPTION
**Pull request by charliegriefer**
Thanks for the assist @briandowns

### CODE REVIEW

**Comment by charliegriefer** [comment]
Added SpeakerID in order to link speaker to presentation. If this isn't OK, let me know.

File: migrations/20161216170348_create_presentations.up.fizz [hunk 1]
```diff
@@ -0,0 +1,6 @@
+create_table("presentations", func(t) {
+	t.Column("title", "string", {})
+	t.Column("short_description", "string", {})
+	t.Column("long_description", "text", {})
+	t.Column("speaker_id", "integer", {})
```
Comment (bketelsen) [inline migrations/20161216170348_create_presentations.up.fizz]: This needs a join table - presentation_speakers

File: models/presentation.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+// Presentation represents a presentation/session
+type Presentation struct {
+	ID               int       `json:"id" db:"id"`
+	CreatedAt        time.Time `json:"created_at" db:"created_at"`
+	UpdatedAt        time.Time `json:"updated_at" db:"updated_at"`
+	Title            string    `json:"title" db:"title"`
+	ShortDescription string    `json:"short_description" db:"short_description"`
+	LongDescription  string    `json:"long_description" db:"long_description"`
+	SpeakerID        int       `json:"speaker_id" db:"speaker_id"`
```
Comment (bketelsen) [inline models/presentation.go]: This shoudl be a slice of speakers, with the join table above

Comment (charliegriefer) [reply models/presentation.go]: If we have the join table, do we even need SpeakerID in the presentation table at all?

Comment (charliegriefer) [reply models/presentation.go]: Ah, never mind. Ignore me :)

The presentation model needs a slice of Speaker IDs. Got it.

Comment (bketelsen) [reply models/presentation.go]: 👍

**Pull request closed without merging**